alves/style-editor.css

```diff
@@ -647,7 +647,7 @@ object {
 
 .wp-block-cover h2,
 .wp-block-cover-image h2 {
-	font-size: 2.592em;
+	font-size: 2.592;
```

Comment:
I see where this is coming from but we should double check that it doesn't break existing websites even though it's the correct syntax.

Reply:
Verified the changes against classes like `.has-larger-font-size` `.has-huge-font-size` and headers in Varia and one of child themes and it seems to be working fine.